crates/storage-api/src/outbox_table/mod.rs

```diff
@@ -62,6 +62,6 @@ pub trait OutboxTable {
     fn truncate_outbox(
```

Comment:
Probably overthinking this but I wonder if `trim_outbox` isn't a better name for operations of this type. Truncate to my mind is more closely associated with completely emptying a data structure, whereas "trim" is often used with logs to drop unwanted prefix data leaving the most recent entries intact.

Reply:
Ah! This is useful context, thanks for the pointer!